docs/migration/5_0.md

```diff
@@ -498,14 +498,23 @@ The following keys have been added to `configurator-common.ts`:
 - Class now implements 'DoCheck'.
 - Added `ChangeDetectionRef` to constructor.
 - Added `KeyValueDiffers` to constructor.
-- Added host attribute `role="alert"`
-- Added new property `hidden` which is host for class `cx-visually-hidden`
-- content rendering depends on `hidden` flag
+- Added host attribute `role="alert"`.
+- Added new property `hidden` which is host for class `cx-visually-hidden`.
+- content rendering depends on `hidden` flag.
 
 ### StockNotificationDialogComponent
 
 #### Translation (i18n) changes
 
 - `subscriptionDialog.notifiedSuffix` changed from `as soons as this product is back in stock.` to `as soon as this product is back in stock.`.
 - `subscriptionDialog.manageChannelsPrefix` changed from `Manage your prefered notification channels at the ` to `Manage your preferred notification channels on the `.
-- `subscriptionDialog.manageSubscriptionsPrefix` changed from `You can manage your subscriptions at ` to `You can manage your subscriptions on `.
\ No newline at end of file
+- `subscriptionDialog.manageSubscriptionsPrefix` changed from `You can manage your subscriptions at ` to `You can manage your subscriptions on `.
+
+### ProductIntroComponent
+
+- Subject `reviewsTabAvailable` has been renamed to "areReviewsAvailable$".
```

Comment:
@pawelfras I think this doc note should be:
Property \`reviewsTabAvailable\` has been renamed to \`areReviewsAvailable$\`.